src/components/Sidebar/SearchResults.js

```diff
@@ -31,8 +36,18 @@ function SearchResults({
       </div>
     );
   }
+  if (searchResults.length === 0 && currentSearchInput.length >= 3) {
+    return (
+      <div className={classes.inputHint}>
+        No results
```

Comment:
Let's use the `Typography` material-ui component here so that we get the styling and font right.